### PR TITLE
Add 'examples' and 'README', remove '.editorconfig' and '.travis.yml'

### PR DESCRIPTION
**Pull request by JunyiYi**
Update the folder structure according to feedback.

### CODE REVIEW

**Review by metacpp — APPROVED** [review]
LGTM but follow the comments in the PR.

**Comment by metacpp** [comment]
@JunyiYi thanks for the updates, please go ahead and publish this new version.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2642 @@
+{
```
Comment (metacpp) [inline package-lock.json]: I don't think we need to check in this file.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,2642 @@
...
+  "name": "generator-az-terra-module",
+  "version": "0.2.1",
```
Comment (metacpp) [inline package-lock.json]: Please update your pacakge.json to `0.3.0`, add git tag, and do publish after merge.

Comment (JunyiYi) [reply package-lock.json]: Will do the publish later to include another change related to LICENSE wizard.

**Pull request merged**